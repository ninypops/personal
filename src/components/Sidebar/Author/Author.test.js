// @flow strict
import React from 'react';
import renderer from 'react-test-renderer';
import Author from './Author';

describe('Author', () => {
  const props = {
    author: {
      name: 'Nina Limbrick',
      photo: '/ninypops_profile.jpg',
      bio: 'Front-end developer using React. Currently navigating a pandemic. (she/they)'
    },
    isIndex: false
  };

  it('renders correctly', () => {
    const tree = renderer.create(<Author {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
