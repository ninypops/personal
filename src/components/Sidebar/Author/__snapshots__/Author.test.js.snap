// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Author renders correctly 1`] = `
<div
  className="author"
>
  <a
    href="/"
  >
    <img
      className="author__photo"
      height="75"
      src="/ninypops_profile.jpg"
      width="75"
    />
  </a>
  <h2
    className="author__title"
  >
    <a
      className="author__title-link"
      href="/"
    >
      Nina Limbrick
    </a>
  </h2>
  <p
    className="author__subtitle"
  >
    Front-end developer using React. Currently navigating a pandemic. (she/they)
  </p>
</div>
`;
